stable/grafana/templates/deployment.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/dashboards-config: {{ include (print $.Template.BasePath "/dashboards-configmap.yaml") . | sha256sum }}
       {{- range $key, $value := .Values.server.annotations }}
```

Comment:
Why can't the user add these K/V pair with `.Values.server.annotations` before creating the chart?